jamf2snipe

```diff
@@ -759,15 +771,16 @@ for jamf_type in jamf_types:
             jamf = search_jamf_asset(jamf_asset['id'])
         elif jamf_type == 'mobile_devices':
             jamf = search_jamf_mobile(jamf_asset['id'])
-        if jamf == None:
+        if not jamf:
             continue
 
         # If the entry doesn't contain a serial, then we need to skip this entry.
-        if jamf['general']['serial_number'] == 'Not Available':
+        if not jamf['general']['serial_number']:
```

Comment:
In my current JAMF instance, a system that is missing info may return null or an empty string ('') which would fail this check - however that fact doesn't seem to be consistent, sometimes it is null, sometimes it is an empty string, hence why I made the modification. I have not ran across the "Not Available" string, this may be an artifact from the old API?

Reply:
It could also be a school.apple.com thing. I'd recommend altering the line to check for for both so if it's either that literal string or null we're covered.